DKV2/csvwriter.cpp

```diff
@@ -5,6 +5,7 @@
 #include "helperfile.h"
 #include "appconfig.h"
 #include "csvwriter.h"
+#include <memory>
```

Comment:
Auch in dieser Datei gibt es keine Verwendung. Ist Dein C++ up to date? Verwendest Du C++ 17?